### PR TITLE
improve team stories, selectionPopup padding

### PR DESCRIPTION
**Pull request by jakob223**
- combine all the different mock stores in team stories so as to reduce code repetition and make it easier to make bigger team stories
 - add stories for the channel and member pages, with and without selections
 - make the selection box take up vertical space on mobile, and put it in a floating box.

Not covered in this PR is scrolling up when making a selection that then becomes hidden; this will require changes to `Kb.SectionList` so if it happens it will be in a separate PR.

cc @keybase/y2ksquad, in particular CC @buoyad @Daniel-Wang @jzila on the storybook changes if we keep this and you make more team stories

### CODE REVIEW

File: shared/__mocks__/feature-flags.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ff: FeatureFlags = {
   stellarExternalPartners: false,
   tabletSupport: false,
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```
Comment (hyperobject) [inline shared/__mocks__/feature-flags.tsx]: probably don't want this turned on in master yet

File: shared/util/feature-flags.native.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const ff: FeatureFlags = {
   stellarExternalPartners: true,
   tabletSupport: true,
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```
Comment (hyperobject) [inline shared/util/feature-flags.native.tsx]: ditto, probably shouldn't flip this on just yet

File: shared/__mocks__/feature-flags.tsx (hunk 1 above)
Comment (jakob223) [reply shared/__mocks__/feature-flags.tsx]: Even for stories? I think it is more useful for stories to capture the new work and changes it has on other new work than to make sure it doesn't impact the old work too much.

File: shared/util/feature-flags.native.tsx (hunk 2 above)
Comment (jakob223) [reply shared/util/feature-flags.native.tsx]: Oops, this one was a mistake

File: shared/__mocks__/feature-flags.tsx (hunk 1 above)
Comment (hyperobject) [reply shared/__mocks__/feature-flags.tsx]: oh whoops misread the file name here, you're right!

File: shared/teams/team/selection-popup.tsx [hunk 3]
```diff
@@ -29,6 +29,12 @@ const tabThings = {
   members: 'member',
 }
 
+// How much to bump the list height by when the selection popup is present on mobile
+const heightsMap = {
+  channels: {height: 70},
```
Comment (buoyad) [inline shared/teams/team/selection-popup.tsx]: It'd be nice to have some dynamic solution here, it's likely that the content won't be static.

Comment (jakob223) [reply shared/teams/team/selection-popup.tsx]: Hmm, as in measuring the floating thing and then plumbing that here? ick that sounds unfun.

Comment (buoyad) [reply shared/teams/team/selection-popup.tsx]: The alternative is more unfun I think

Comment (buoyad) [reply shared/teams/team/selection-popup.tsx]: I'd say we could defer this to another ticket, but my members PR makes this dynamic already :/

Comment (jakob223) [reply shared/teams/team/selection-popup.tsx]: Okay, I'll do it here when your PR is merged.

File: shared/teams/main/team-row.stories.tsx [hunk 4]
```diff
@@ -1,27 +1,14 @@
...
-      ['2', Constants.makeTeamMeta({isOpen: true, memberCount: 11947, teamname: 'fan_club'})],
-      ['3', Constants.makeTeamMeta({isOpen: false, memberCount: 234, teamname: 'club_penguin'})],
-    ]),
-  }
-})
+import {fakeTeamIDs, store} from '../stories'
 
 const load = () =>
   Sb.storiesOf('Teams', module)
     .addDecorator(story => <Sb.MockStore store={store}>{story()}</Sb.MockStore>)
```
Comment (thebearjew) [inline shared/teams/main/team-row.stories.tsx]: @jakob223 I wrote a thing to help with this: `Sb.updateStoreDecorator`

https://github.com/keybase/client/blob/master/shared/stories/storybook.shared.tsx#L107-L110

**Pull request merged**